api/audit/models.py

```diff
@@ -44,10 +54,10 @@ class AuditLog(LifecycleModel):
     related_object_type = models.CharField(max_length=20, null=True)
     related_object_uuid = models.CharField(max_length=36, null=True)
 
-    skip_signals = models.CharField(
+    skip_subscribers = models.CharField(
```

Comment:
I am still a bit uncertain about this name (and the way it used) Is this a case of YAGNI(maybe we just need a boolean called `environment_updated`?) what do you think?

Reply:
We could just be more explicit and call it `skip_signals_and_hooks` or something (with the above caveat that the column name has to remain the same)? I think it gives us a good amount of flexibility so I'm keen to keep it how it is.